packages/storage-web-api/src/request-validators/get-website-request-validator.ts

```diff
@@ -1,10 +1,32 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import { injectable } from 'inversify';
-import { ApiRequestValidator } from 'service-library';
+import { Context } from '@azure/functions';
```

Comment:
Not sure we need to have full azure func context dependencies here. Will this if we pass just rest request values to be more simple like scanId, websiteId, etc.